spec/routes/api/project_spec.rb

```diff
@@ -2,28 +2,22 @@
 
 require_relative "spec_helper"
 
-RSpec.describe Clover, "vm" do
+RSpec.describe Clover, "project" do
   let(:user) { create_account }
 
   let(:project) { user.create_project_with_default_policy("project-1") }
 
   describe "unauthenticated" do
-    it "not list" do
-      get "/api/project"
-
-      expect(last_response).to have_api_error(401, "Please login to continue")
-    end
-
-    it "not create" do
-      post "/api/project"
-
-      expect(last_response).to have_api_error(401, "Please login to continue")
-    end
-
-    it "not delete" do
-      delete "api/project/#{project.ubid}"
-
-      expect(last_response).to have_api_error(401, "Please login to continue")
+    it "cannot perform authenticated operations" do
```

Comment:
I know we use this style in a lot of tests.  I don't think we should change it in this PR, but what are your thoughts on restructuring this and similar tests from:

```ruby
  it desc do
    array.each{|*args| do_something(*args)}
  end
```

to:

```ruby
  array.each do|*args| 
    it "#{desc} with #{args.inspect}" do
      do_something(*args)}
    end
  end
```

The proposed approached is slightly slower, but it has the following advantages:

* Makes each test case independent (separate transaction for each)
* Shows exactly which test cases fail (accelerates debugging if the failure only occurs in a subset of cases and not all cases)

Reply:
Several times, I enable cases individually to identify the ones that fail